django_project/core/urls.py

```diff
@@ -32,7 +33,11 @@ def handler500(request):
     }))
 
 
-urlpatterns = []
+urlpatterns = [
+    # force redirect to /en/
+    # to fix 500 error temporarily  after user logged in on production
+    url(r'^$', RedirectView.as_view(url='/en/')),
```

Comment:
Can you try the following:

Replace this with:

```suggestion
    url(r'^$', index_view),
```

Where `index_view` defined as

```python
from django.utils import translation
from django.shortcuts import redirect
from django.urls import reverse

def index_view(request):
    user_language = translation.get_language() or settings.LANGUAGE_CODE or 'en'
    translation.activate(user_language)
    # I looked up the view name in base.urls
    return redirect(reverse('home'))
```

Reply:
It might not work out of the box, but try and modify if possible.

Reply:
Thanks @lucernae, I implement your code in the new commit. I gave it a try on my local machine and it worked as expected.